lib/debug/breakpoint.rb

```diff
@@ -328,12 +328,15 @@ def to_s
   end
 
   class WatchIVarBreakpoint < Breakpoint
-    def initialize ivar, object, current
+    def initialize ivar, object, current, cond: nil, command: nil
       @ivar = ivar.to_sym
       @object = object
-      @key = [:watch, @ivar].freeze
+      @key = [:watch, object.object_id, @ivar].freeze
```

Comment:
WatchBreakpoint could be initialized for multiple instances, so we should take that into consideration when filtering duplicated bps.